enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -45,15 +50,15 @@ <h3 class="course-confirmation-title">{{ confirmation_text }}</h3>
             <span>{{ starts_at_text }} {{ course_start_date }} &nbsp;| &nbsp; {{ course_pacing }} - {{ course_pace_text }}</span>
           </div>
           {{ course_short_description }}
-          <a class="text-underline" href="#!">{{ view_course_details_text }}</a>
+          <a id="view-course-details-link" class="text-underline" href="#!">{{ view_course_details_text }}</a>
         </div>
         <form>
           <div class="caption">{{ select_mode_text }}</div>
           {% for course_mode in course_modes %}
           <div class="radio-block">
             <div class="radio">
               {% if course_modes|length > 1 %}
-                <input type="radio" name="data_sharing_consent" id="radio${{ forloop.counter0 }}" {% if forloop.first %}checked="checked"{% endif %} />
+                <input type="radio" name="data_sharing_consent" id="radio{{ forloop.counter0 }}" {% if forloop.first %}checked="checked"{% endif %} />
```

Comment:
@replaceafill Did you check whether these IDs are referenced somewhere else? Just asking because if they are, removing `$` might break existing functionality.